src/consent-manager/container.tsx

```diff
@@ -23,6 +23,7 @@ interface ContainerProps {
   resetPreferences: () => void
   closeBehavior?: CloseBehavior
   destinations: Destination[]
+  customCategories?: CustomCategories | undefined
```

Comment:
Doesn't the `?` take care of the undefined case here?

Reply:
I think I originally had it as just `customCategories?: CustomCategories` but it was failing validation - because sometimes we do pass in customCategories but the value is undefined.